### ISSUE TITLE
Buttons must have discernible text

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#216]
Source: `<button aria-label="" class="content--slotted icon-start-empty icon-end-empty" type="button"><span class="content"><slot></slot></span></button>`
Element Location: [".c-propertyContainer__contentCombo > calcite-button[color=\"neutral\"][appearance=\"transparent\"][width=\"auto\"]",".icon-start-empty.content--slotted.icon-end-empty"]
failureSummary: Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"Tags: [Ljava.lang.String;@52d239ba
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
